wikiman.sh

```diff
@@ -285,6 +296,23 @@ while getopts l:s:H:pqhRSk o; do
 done
 shift "$((OPTIND - 1))"
 
+# Dependency check
+
+echo 'Checking dependencies...'
+for dependency in 'man' 'fzf' 'rg' 'awk' 'w3m' "$conf_fuzzy_finder"; do
+	which "$dependency" >/dev/null || {
+		echo "Error: missing dependency: cannot find $dependency executable!"
+		exit 1
+	}
+done
```

Comment:
- Printing 'Checking dependencies...' every time is too verbose.
- Dependencies should be in an array defined beforehand.
- Style should be changed - see commend for lines 311-314.

Reply:
Do you want the exit code in this block set to `127`?

Reply:
>      Printing 'Checking dependencies...' every time is too verbose.

This only get printed 1 time before the dependency check start. I don't understand exactly what I you want. Do you want it to be removed completely?